### ISSUE TITLE
Sitewide

### ISSUE THREAD

**Issue by ajnyga** [pkp/oaiJats#6]
Hi,

Maybe this and the other plugin connected to this one could be sitewide?

**Comment by asmecher** [comment]
Perhaps as an override; one of the use cases it's being developed for requires it to be selectively enabled (for large installations, one or two of whose journals need to be indexed via JATS/OAI.

**Comment by asmecher** [comment]
This is for [Coalition Publi.ca](https://www.coalition-publi.ca/) -- most of the journals included there will use a combination of the OAIJats plugin and the [JATS Template plugin](https://github.com/pkp/jatsTemplate) (although some may generate the JATS through another means).

The OAIJats plugin already overrides the front content with what's built into OJS; I think that might be a useful starting point? I envision custom JATS requirements as some kind of filter process, so it might be worth adding a hook or two here so that aspects of the document can be overridden before output.

**Comment by ajnyga** [comment]
Ah, ok great!

I think they are mainly interested in the article access status, possible embargo dates and funding data. It could also be that they need a metadataformat with a specific name, I will ask about that next week.

**Comment by odolany** [comment]
Do I understand correctly from the above that accessing all multi-journal contents through /index.php/index/oai?verb=ListRecords&metadataPrefix=jats is not to be expected by design?

**Comment by asmecher** [comment]
@odolany, yes for the moment -- though we're open to proposals/contributions. It would still need to be able to operate on a journal-by-journal basis but it may be possible to support both.
